stdlib/public/runtime/ImageInspectionCommon.h

```diff
@@ -51,7 +51,7 @@
 
 namespace swift {
 struct MetadataSections;
-static constexpr const uintptr_t CurrentSectionMetadataVersion = 2;
+static constexpr const uintptr_t CurrentSectionMetadataVersion = 3;
```

Comment:
Bumping this version number is critical as it's the only way downstream components will be able to tell if a `MetadataSections` structure includes the new field.

Reply:
Thanks!

Reply:
```/Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift/stdlib/public/runtime/RuntimeAttribute.cpp:18:10: fatal error: 'Lazy.h' file not found
#include "Lazy.h"
         ^~~~~~~~
```

Reply:
Fixed.